taxi/urls.py

```diff
@@ -46,6 +49,21 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path(
+        "driver/driver-create/",
+        DriverCreateView.as_view(),
+        name="driver-create"
+    ),
+    path(
+        "driver/<int:pk>/driver-update/",
```

Comment:
Consider changing the path 'driver/<int:pk>/driver-update/' to 'drivers/<int:pk>/update/' to maintain consistency with the other driver-related URLs.